plugins/generator-1.20.4/neoforge-1.20.4/templates/item/item_container.java.ftl

```diff
@@ -33,49 +33,29 @@
 package ${package}.item.inventory;
 
 <#compress>
-@Mod.EventBusSubscriber(Dist.CLIENT) public class ${name}InventoryCapability implements ICapabilitySerializable<CompoundTag> {
+@Mod.EventBusSubscriber(Dist.CLIENT) public class ${name}InventoryAttachment extends ItemStackHandler {

```

Comment:
This is not attahcment but capability. You are confusing the two terms. Please keep the original name.

This class describes functionality, not storage, so it is capability

Reply:
I'd say it now works as attachment but is also used as capability.